bitbots_blackboard/bitbots_blackboard/capsules/pathfinding_capsule.py

```diff
@@ -36,38 +52,67 @@ def __init__(self, blackboard: "BodyBlackboard", node: Node):
             ['controller.orient_to_goal_distance'])["controller.orient_to_goal_distance"]
 
     def publish(self, msg: PoseStamped):
+        """
+        Sends a goal to the pathfinding.
+        """
         self.goal = msg
         self.pathfinding_pub.publish(msg)
 
     def get_goal(self) -> PoseStamped:
+        """
+        Returns the latest goal that was send to the pathfinding.
+        """
         return self.goal
 
     def cancel_goal(self):
+        """
+        This function cancels the current goal of the pathfinding,
+        which will stop sending cmd_vel messages to the walking.
+        This does not stop the walking itself.
+        """
         self.pathfinding_cancel_pub.publish(Empty())
 
     def cmd_vel_cb(self, msg: Twist):
         self.current_cmd_vel = msg
 
+    def get_current_cmd_vel(self) -> Twist:
+        """
+        Returns the latest cmd_vel message that was send to the walking.
+
+        This message can be from the behavior or from the pathfinding.
+        """
+        return self.current_cmd_vel
+
     def stop_walk(self):
+        """
+        This function stops the walking. It does not cancel the current goal of the
+        pathfinding and the walking will start again if the pathfinding sends a new message.
+        """
         # send special command to walking to stop it
         msg = Twist()
         msg.angular.x = -1.0
         self.direct_cmd_vel_pub.publish(msg)
 
     def calculate_time_to_ball(self):
+        """
+        Calculates the time to ball and saves it in the team data capsule.
+        """
         # only send new request if previous request is finished or first update
         # also verify that the ball and the localization are reasonably recent/accurate
         if self._blackboard.world_model.ball_has_been_seen() and \
                 self._blackboard.world_model.localization_precision_in_threshold():
```

Comment:
todo don't check that

Reply:
This will be done in a different changeset. @Flova please create an issue.